### PR TITLE
[TACHYON-1035] Fix lsTest to use getFileInfoList

### PR DESCRIPTION
**Pull request by gautam-kumar**
https://tachyon.atlassian.net/browse/TACHYON-1035

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java [hunk 1]
```diff
@@ -643,27 +643,25 @@ public void listFilesTest() throws InvalidPathException, FileDoesNotExistExcepti
...
-  //  Assert.assertEquals(111, mMasterInfo.ls(new TachyonURI(TachyonURI.SEPARATOR), true).size());
-  //}
+    Assert.assertEquals(1,
+      mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/i0/j0"))).size());
+    for (int i = 0; i < 10; i ++) {
+      Assert.assertEquals(10,
+        mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/i" + i))).size());
+    }
+    Assert.assertEquals(10,
+      mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/"))).size());
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java]: I think this needs more indentation.

File: integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java [hunk 2]
```diff
@@ -643,27 +643,25 @@ public void listFilesTest() throws InvalidPathException, FileDoesNotExistExcepti
...
-  //    Assert.assertEquals(11, mMasterInfo.ls(new TachyonURI("/i" + i), true).size());
-  //  }
-  //  Assert.assertEquals(11, mMasterInfo.ls(new TachyonURI(TachyonURI.SEPARATOR), false).size());
-  //  Assert.assertEquals(111, mMasterInfo.ls(new TachyonURI(TachyonURI.SEPARATOR), true).size());
-  //}
+    Assert.assertEquals(1,
+      mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/i0/j0"))).size());
+    for (int i = 0; i < 10; i ++) {
+      Assert.assertEquals(10,
+        mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/i" + i))).size());
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java]: I think this needs more indentation.

File: integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java [hunk 3]
```diff
@@ -643,27 +643,25 @@ public void listFilesTest() throws InvalidPathException, FileDoesNotExistExcepti
...
-  //  Assert.assertEquals(1, mMasterInfo.ls(new TachyonURI("/i0/j0"), true).size());
-  //  for (int i = 0; i < 10; i ++) {
-  //    Assert.assertEquals(11, mMasterInfo.ls(new TachyonURI("/i" + i), false).size());
-  //    Assert.assertEquals(11, mMasterInfo.ls(new TachyonURI("/i" + i), true).size());
-  //  }
-  //  Assert.assertEquals(11, mMasterInfo.ls(new TachyonURI(TachyonURI.SEPARATOR), false).size());
-  //  Assert.assertEquals(111, mMasterInfo.ls(new TachyonURI(TachyonURI.SEPARATOR), true).size());
-  //}
+    Assert.assertEquals(1,
+      mFsMaster.getFileInfoList(mFsMaster.getFileId(new TachyonURI("/i0/j0"))).size());
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java]: I think this needs more indentation.

**Comment by gpang** [comment]
I think some indentation issues are failing the build. Thanks!

**Comment by gautam-kumar** [comment]
How do you do style check?

**Comment by gpang** [comment]
LGTM

**Pull request merged**